auth-web/src/components/auth/staff/admin/AdministrativeBN.vue

```diff
@@ -2,14 +2,14 @@
   <v-card id="administrative-bn" class="pa-8" flat>
     <div class="view-header flex-column">
       <v-row no-gutters>
-          <v-col col="6">
-            <h2 class="view-header__title">Administrative BN</h2>
-          </v-col>
-          <v-col col="6" class="pr-0" align="right">
-            <v-btn link color="primary" v-if="businessDetails" @click="resetSearch()">Search another Business</v-btn>
-            <v-btn link class="ml-2" v-if="businessDetails" @click="reload()">Reload</v-btn>
-          </v-col>
-        </v-row>
+        <v-col col="6">
+          <h2 class="view-header__title">Administrative BN</h2>
+        </v-col>
+        <v-col col="6" class="pr-0" align="right">
```

Comment:
The spacing problem is annoying, I'm having the same issue with this project - hoping to get some eslint rules so formatting is easier to manage.